service/pom.xml

```diff
@@ -280,6 +280,23 @@ under the License.
                     </dependency>
                 </dependencies>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
```

Comment:
will the resulting artifact have all dependencies inside? let make sure it does not -- typically "library jar" is meant to have only module's classes, without deps.